OOP-War/Models/Game.cs

```diff
@@ -0,0 +1,22 @@
+namespace OOP_War.Models;
```

Comment:
We discussed that projects, folders and namespaces should be named using PascalCase